### DISCUSSION TITLE
add always below other windows option

### DISCUSSION THREAD

**Discussion by florianfelix** [tauri-apps/tauri#1832] [Ideas]
If it is not to complicated a window.set_always_on_bottom or somesuch option would be appreciated.

**Comment by amrbashir** [comment]
can your provide a use case for this feature ? just wanna understand the full picture.

**Reply by florianfelix** [reply]
The use case i had in mind was writing an alternative for conky.
Also it would be just more complete :)

**Reply by amrbashir** [reply]
Sure why not, can you open an issue https://github.com/tauri-apps/tao and once we implement it there we can add it to tauri.

**Comment by apacheli** [comment]
I see that the issue was marked as completed at https://github.com/tauri-apps/tao/pull/522, however, it does not seem to be supported yet according to https://github.com/tauri-apps/tao/issues/470 if I'm not mistaken. Our use case is that we want to simulate widgets on the desktop where they should never be on top of other windows.

**Reply by FabianLars** [reply]
470 is basically irrelevant, but apart from that what it says is actually that `tao` supports it but `winit` doesn't. We're tracking the differences because we want to move back to winit in the near future.

p.s. tao support doesn't mean that tauri supports something too. In this case it will still be a while until tauri will expose it.
